api/src/main/java/org/pmiops/workbench/api/InstitutionController.java

```diff
@@ -29,11 +31,24 @@
   @Override
   @AuthorityRequired({Authority.INSTITUTION_ADMIN})
   public ResponseEntity<Void> deleteInstitution(final String shortName) {
-    if (institutionService.deleteInstitution(shortName)) {
-      return ResponseEntity.noContent().build();
-    } else {
-      throw new NotFoundException(String.format("Could not delete Institution %s", shortName));
+    final DeletionResult result = institutionService.deleteInstitution(shortName);
+
+    // I wanted to use a switch here but Java complained about lacking a return value
+    if (result == DeletionResult.HAS_VERIFIED_AFFILIATIONS) {
+      // TODO: 405 or 409?
```

Comment:
I chose 409 but I'm open to discussion here - will remove the comments when we come to consensus